tests/testthat/test-test.R

```diff
@@ -161,7 +161,7 @@ test_that("app template works with runTests", {
     )
   ))
 
-  lapply(combos, function(combo) {
+  for (combo in combos) {local({
```

Comment:
Just curious: why the change?

Reply:
`testthat` says that the failure occurs at line 164 and not 172 or the actual failure line